Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest as base
+
+RUN microdnf update -y && \
+    microdnf install -y \
+        python3-devel gcc git python-pip && \
+    pip install --upgrade --no-cache-dir pip wheel && \
+    microdnf clean all
+
+FROM base as builder
+WORKDIR /build
+
+RUN pip install --no-cache tox
+COPY README.md .
+COPY pyproject.toml .
+COPY tox.ini .
+COPY caikit_computer_vision caikit_computer_vision
+# .git is required for setuptools-scm get the version
+RUN --mount=source=.git,target=.git,type=bind \
```

Comment:
Good question! I think yes, but I would like to not block this PR / the SDXL PR on it if it's alright with you, since I'm about to be OOO for a while, and I would prefer to give people an image built using stuff off of `main` if possible 😄